elements/pfe-button/src/pfe-button.json

```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Button",
+  "description": "",
+  "type": "object",
+  "tag": "pfe-button",
+  "class": "PfeButton",
+  "category": "content",
+  "properties": {
+    "slots": {
+      "title": "Slots",
+      "description": "Definition of the supported slots",
+      "type": "object",
+      "properties": {
+        "default": {
+          "title": "Default slot",
+          "type": "array",
+          "namedSlot": false,
+          "items": {
+            "oneOf": [
+              {
+                "$ref": "button"
+              }
+            ]
+          }
+        }
+      },
+      "required": ["default"]
+    },
+    "attributes": {
+      "title": "Attributes",
+      "type": "object",
+      "properties": {
+        "variant": {
+          "title": "Style variant",
+          "type": "string",
+          "prefixed": true,
+          "enum": ["primary", "secondary", "tertiary", "danger", "control"]
+        }
+      }
+    },
+    "events": {
+      "title": "Events",
+      "description": "Definition of the supported events",
+      "type": "object",
+      "properties": {}
```

Comment:
Can you add the click event to the schema here too?

Reply:
👍 done!